ssz/src/main/java/org/ethereum/beacon/ssz/SSZHasher.java

```diff
@@ -0,0 +1,61 @@
+package org.ethereum.beacon.ssz;
+
+import net.consensys.cava.bytes.Bytes;
+import java.util.function.Function;
+
+/**
+ * <p>SSZ Hashing helper made according to the following specs:
+ * <a href="https://github.com/ethereum/eth2.0-specs/blob/master/specs/simple-serialize.md#tree-hash">SSZ Tree Hash</a></p>
+ *
+ * <p>It's based on {@link SSZSerializer}, so for setup help, check its Javadoc
+ * and {@link SSZSerializerBuilder} documentation</p>
+ */
+public class SSZHasher {
```

Comment:
What do you think about adding a parameter that specifies a type of the output hash value?

Reply:
done